src/keria/app/cli/keria.py

```diff
@@ -22,8 +17,7 @@ def main():
         return
 
     try:
-        doers = args.handler(args)
-        directing.runController(doers=doers, expire=0.0)
```

Comment:
I removed this call because the handlers themselves to not return any `doers`, they are running them themselves.

Reply:
agree https://github.com/WebOfTrust/keria/blob/4981d84a0e6d4536d2a088c1ffdd47871c253799/src/keria/app/cli/commands/start.py#L114

Reply:
I concur, this makes sense.